sharedresource/v1alpha1/types_share.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright The OpenShift authors.
```

Comment:
this is an unusual header

Reply:
removed in next push

Reply:
Still applies.

Reply:
I had not pushed changes yet, will soon.